build/grunt.js

```diff
@@ -31,7 +32,7 @@ module.exports = function(grunt) {
       release: {
         tag_name: 'v'+ version.full,
         name: version.full,
-        body: chg.find(version.full).changesRaw
+        body: nodeCli.exec('conventional-changelog', '-p videojs', {silent: true}).output
```

Comment:
That's a valid concern. The repo is available [here](https://github.com/nzakas/shelljs-nodecli), I think it's just missing from the package.json. Also, nicholas has a good track record of maintaining modules or giving it away to other maintainers to help out (see eslint as an example). In addition, I think at this point `shelljs-nodecli` is feature complete and there isn't much necessary to do with it; for example, it hasn't been updated in 3 years but still works great.
In addition, it will instead encapsulate the whole thing of running a node binary easily, so, we don't have to worry about how to run it properly or get to the binary, we only need to think about how to use `conventional-changelog`.
Hopefully, this addresses your concerns.